Raspi/Test/DataToTempCSV.py

```diff
@@ -0,0 +1,92 @@
+import json
+from datetime import datetime
+import os
+import csv
+
+
+class DataToTempCSV:
```

Comment:
This could be written as a function instead as you're not keeping the class instance when you call it

Reply:
It might save you a lot of trouble finding the temp files again if the function returned the temp file name.

Reply:
The temp files are always stored in the directory where the script is run. For example when the script is run in /Das/Test it will be saved in the directory